bugbot/rules/has_str_no_range.py

```diff
@@ -109,7 +109,7 @@ def get_bz_params(self, date):
             "n6": 1,
             "f6": "reporter",
             "o6": "regexp",
-            "v6": r"^.*@softvision\.(ro|com)$",
+            "v6": r"^.*@mozilla\.com$",
```

Comment:
What do you think about the approach mentioned in https://github.com/mozilla/bugbot/issues/2338#issuecomment-1930646548?